MauiProject/MauiProject.csproj

```diff
@@ -49,8 +49,8 @@
 	</ItemGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="7.0.0" />
-		<PackageReference Include="Syncfusion.Maui.Core" Version="20.4.51" />
+		<PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="9.0.0" />

```

Comment:
Microsoft.Maui.controls package is missing? Please ensure it's whether needed or not